apps/doji-frontend/src/pages/reset-password/[token].tsx

```diff
@@ -0,0 +1,79 @@
+import RegisteredTextfield from '@frontend/components/Register/registerTextfield'
+import { httpClient } from '@frontend/services'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { IUserResetPasswordRequest } from '@libs/api'
+import { Button, Stack } from '@mui/material'
+import router, { useRouter } from 'next/router'
+import * as yup from 'yup'
+
+import { SubmitHandler, useForm } from 'react-hook-form'
+import toast from 'react-hot-toast'
+import { useMutation } from 'react-query'
+
+const ResetPasswordSchema = yup.object({
+  newPassword: yup
+    .string()
+    .required('enter new password')
+    .min(8, 'Must have at least 8 characters'),
+  confirmation: yup
```

Comment:
```suggestion
  confirmPassword: yup
```

confirmation ความหมายกว้างไป

Reply:
merged to dev, added mutation things, change to more appropriate variable names.